packages/manager/modules/pci/src/projects/project/vouchers/credit/credit-agora.controller.js

```diff
@@ -30,29 +32,30 @@ export default class CloudProjectBillingVouchersAddcreditAgoraCtrl {
   $onInit() {
     this.amount = 10;
     this.loading = true;
-    return this.$http.get('/order/catalog/formatted/cloud', {
+    return this.$http.get('/order/catalog/public/cloud', {
```

Comment:
Could we replace this by an `ovh-api-services` call ?

Reply:
This is fixed. Thanks :)